cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/get_compute_fleet_status.erb

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+cat << EOF > get-compute-fleet-status.py
+import json
+import logging
+
+import argparse
+import boto3
+
+DB_KEY = "COMPUTE_FLEET"
+DB_DATA = "Data"
+
+COMPUTE_FLEET_STATUS_ATTRIBUTE = "status"
+COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE = "lastStatusUpdatedTime"
+
+log = logging.getLogger()
+
+
+def get_status_with_last_updated_time(table_name, region):
+    """Get compute fleet status and the last compute fleet status updated time."""
+    try:
+        table = boto3.resource("dynamodb", region_name=region).Table(table_name)
+        compute_fleet_item = table.get_item(ConsistentRead=True, Key={"Id": DB_KEY})
+        if not compute_fleet_item or "Item" not in compute_fleet_item:
+            raise Exception("COMPUTE_FLEET data not found in db table")
+        compute_fleet_status_attribute = compute_fleet_item["Item"].get(DB_DATA).get(COMPUTE_FLEET_STATUS_ATTRIBUTE)
+        compute_fleet_last_updated_time_attribute = compute_fleet_item["Item"].get(DB_DATA).get(COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE)
+        print(json.dumps({COMPUTE_FLEET_STATUS_ATTRIBUTE: compute_fleet_status_attribute, COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE: compute_fleet_last_updated_time_attribute}, sort_keys=True, indent=4))
+
+    except Exception as e:
+        log.warning(
+            "Failed when retrieving fleet status from DynamoDB with error %s. "
+            "This is expected if cluster creation/deletion is in progress",
+            e,
```

Comment:
what about returning and `UNKNOWN` value in case of error?

Reply:
As we discussed, changed to not return in case of error. Errors go to stderror